### ISSUE TITLE
mem (RAM) strange values

### ISSUE THREAD

**Issue by rokobuljan** [sebhildebrandt/systeminformation#789]
**Describe the bug**
Strange RAM memory value for `used`.  
I'm trying to calculate the RAM usage for a hardware and I was perplexed why the RAM usage % was so high.  

**To Reproduce**

Here I'm using a log for `si.mem` and immediately a Linux's `free -b` , and here are the two results: 

```sh
node backend/test.js backend/system.js getMemory && free -b
{
  total: 7910309888,
  free: 1046216704,
  used: 6864093184,
  active: 2942849024,
  available: 4967460864,
  buffers: 456417280,
  cached: 4368478208,
  slab: 341499904,
  buffcache: 5166395392,
  swaptotal: 2147479552,
  swapused: 3670016,
  swapfree: 2143809536
}
               total        used        free      shared  buff/cache   available
Mem:      7910309888  1844736000  1056526336   777359360  5009047552  4977770496
Swap:     2147479552     3670016  2143809536
```

The `used` are nowhere near:

```
6864093184     < si.mem.used
1844736000     < free -b <used>
```

**Current Output**

The logic:

```txt
percent = used * 100 / total
```

To get percentage, if we use the Linux's tool `free -b` we get:

```txt
23.320654008744697 = 1844736000 * 100 / 7910309888
```

whilst by using `mem`'s `used` and `total` values we get this high percentage

```txt
86.77401114731146 = 6864093184 * 100 / 7910309888
```

**Expected behavior**
86.77 is nowhere near 23.32 %  

**Environment (please complete the following information):**
 - systeminformation package version: ^5.17.12
 - OS: Linux (Ubuntu 22.04)
 - Hardware: Intel NUC11TNHi3

**Additional context**

Whould I use `free * 100 / total` instead? 

I am missing the difference in the values from `free -b` (bytes) gives and the ones systeminformation.mem returns

**Comment by sebhildebrandt** [comment]
@rokobuljan ... I think the calculation is correct:

we got total = 7910309888
and free = 1046216704

So the overall used (actively, buff/cache, ...) is 6864093184

So the value of (6864093184 / 7910309888) * 100 = 86,64 is correct. 

To be more specific:
In the `free` command

- `used` is calculated by total - available
- and `available` is an estimation of how much memory is available for starting new applications, without swapping. 
- so the `used` value of free is not really the "used" memory, it is only the one which is kind of the not all reclaimable memory slabs.
- more accurate to calculate used memory is `total` - `free` ... which is used here in this package.

Hope that helps. Closing it for now,
